packages/documentation-site/patternfly-docs/content/training/trainingCard/trainingCard.css

```diff
@@ -1,18 +1,18 @@
-.pf-v5-c-card:not(.pf-org__card-small).pf-org__card {
```

Comment:
I wonder if we could get rid of some of these styles. If not, we should probably tokenify these styles.

Reply:
Followup #4023